Scripts/Python/Ahnonay.py

```diff
@@ -81,10 +82,26 @@ def OnServerInitComplete(self):
         owner = None
         myID = str(PtGetClientIDFromAvatarKey(PtGetLocalAvatar().getKey()))
 
+        ageStruct = ptAgeInfoStruct()
+        ageStruct.setAgeFilename("AhnonayCathedral")
+        ageLinkNode = agevault.getSubAgeLink(ageStruct)
+        if ageLinkNode:
+            localCathedralGuid = ageLinkNode.getAgeInfo().getAgeInstanceGuid()
+
+        folder = vault.getAgesIOwnFolder()
+        contents = folder.getChildNodeRefList()
+        for content in contents:
+            link = content.getChild()
+            link = link.upcastToAgeLinkNode()
+            if link:
+                info = link.getAgeInfo()
+                if info.getAgeFilename() == "AhnonayCathedral":
+                    personalCathedralGuid = info.getAgeInstanceGuid()
+                    break
```

Comment:
```suggestion
        cathedralInfoTemplate = ptVaultAgeInfoNode(0)
        cathedralInfoTemplate.setAgeFilename("AhnonayCathedral")
        if cathedralInfo := folder.findNode(cathedralInfoTemplate):
            personalCathedralGuid = cathedralInfo.upcastToAgeInfoNode().getAgeInstanceGuid()
        else:
            personalCathedralGuid = None
```

Yeah, Cyan's python scripters really didn't know how to write good code for their own engine. Sad.

Reply:
Wait, don't use this suggestion. Unfortunately, `ptVaultNode.findNode()` won't search down the tree the same way the C++ version will. I'm going to leave this suggestion in, though, for future reference.

Reply:
I've updated this suggestion to function properly when #1143 is merged.

Reply:
Changes made.